### PR TITLE
docs(theme): improve cf faq

### PR DESCRIPTION
**Pull request by lixiang810**

### CODE REVIEW

**Comment by Mister-Hope** [comment]
Could you update en pages as well?

**Comment by lixiang810** [comment]
> Could you update en pages as well?

OK, I will do it.

File: docs/theme/src/faq.md [hunk 1]
```diff
@@ -71,11 +71,11 @@ Don't worry, due to network cache, a hot reload when editing markdown files is s
 
 This error indicates that you have an SSR mismatch, and it should not be a problem with theme.
 
-Please check if you are using CloudFrame related services first, if so, make sure you turn off static resource compression.
+Please check if you are using CloudFlare related services first, if so, make sure you turn off static resource compression. Visit [dash.cloudflare.com](https://dash.cloudflare.com), go to Websites-(your domain)-Speed-Optimization, turn `JavaScript` and `HTML` off in `Auto Minify` options.
```
Comment (Mister-Hope) [inline docs/theme/src/faq.md:74]:
```suggestion
Please check if you are using CloudFlare related services first, if so, make sure you turn off static resource compression. Visit [dash.cloudflare.com](https://dash.cloudflare.com), go to Websites → (your domain) → Speed → Optimization, turn `JavaScript` and `HTML` off in `Auto Minify` options.
```

File: docs/theme/src/zh/faq.md [hunk 2]
```diff
@@ -75,11 +75,11 @@ category:
 
 这个错误表明你遇到了 SSR 错配，而且这应该不是主题的问题。
 
-请首先检查你是否在使用 CloudFrame 相关服务，如果有，请确保你关闭静态资源压缩。
+请首先检查你是否在使用 CloudFlare 相关服务，如果有，请确保你关闭静态资源压缩。方法：[dash.cloudflare.com](https://dash.cloudflare.com)-网站-（域名）-速度-优化- Auto Minify，关闭 JavaScript 和 HTML 即可。
```
Comment (Mister-Hope) [inline docs/theme/src/zh/faq.md:78]:
```suggestion
请首先检查你是否在使用 CloudFlare 相关服务，如果有，请确保你关闭静态资源压缩。方法：[dash.cloudflare.com](https://dash.cloudflare.com) → 网站 → `域名` → 速度 → 优化 → Auto Minify，关闭 JavaScript 和 HTML 即可。
```

**Review by Mister-Hope — CHANGES_REQUESTED** [review]
Small updates

**Comment by Mister-Hope** [comment]
Thanks for fixing this❤️

**Comment by lixiang810** [comment]
Also thanks for your developing of theme and plugins, which makes me possible to blog in vuepress 2.

**Pull request merged**